Plugin/Profiling/EventManagerPlugin.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace JustBetter\Sentry\Plugin\Profiling;
+
+use JustBetter\Sentry\Model\SentryPerformance;
+use Magento\Framework\Event\ConfigInterface;
+use Magento\Framework\Event\ManagerInterface;
+use Sentry\Tracing\SpanContext;
+
+class EventManagerPlugin
+{
+    public function __construct(
+        private ConfigInterface $config,
+        private array $excludePatterns = []
+    ) {
+        $this->excludePatterns = array_merge([
```

Comment:
i think the excluded events are self-explained. 
these events happens very often on each request. 
And because the count of spans in a transaction is limited, we have to exclude a few events, which also creates spans in a high amount.